src/UseCases/AddDonation/AddDonationRequest.php

```diff
@@ -174,4 +174,22 @@ public function setAmountFromString( string $value, string $locale = 'de_DE' ) {
 		);
 	}
 
+	public function getPreferredValue( array $values ) {
```

Comment:
These methods don't belong in this value object. You can tell by them not accessing anything in here. They could just as well be static. I'll merge now and submit a follow up to make them static, so it' at least more clear they are not really part of this class.